src/proformbike.cpp

```diff
@@ -1199,7 +1199,7 @@ void proformbike::stateChanged(QLowEnergyService::ServiceState state) {
                 &proformbike::descriptorWritten);
 
         // ******************************************* virtual bike init *************************************
-        if (!firstStateChanged && !virtualBike
+        if (!firstStateChanged && !this->hasVirtualDevice()
 #ifdef Q_OS_IOS
 #ifndef IO_UNDER_QT
```

Comment:
I'm wondering why some have this iOS extra and some don't.

Reply:
all the bikes should have this extra iOS information. This is in order to fix a QT bug about the UUID of bluetooth (one of the very first bugs that I discovered #71

Reply:
Addressed in #1222